eagleEvents/templates/event.html.j2

```diff
@@ -20,32 +32,89 @@
                         </tr>
                     </thead>
                     <tbody>
-                        <tr>
-                            <td>Event 1</td>
-                            <td>Eva</td>
-                            <td>04/12/2018</td>
-                        </tr>
-                        <tr>
-                            <td>Event 2</td>
-                            <td>Jacie</td>
-                            <td>08/22/2018</td>
-                        </tr>
-                        <tr>
-                            <td>Event 3</td>
-                            <td>David</td>
-                            <td>01/23/2019</td>
+                        {% for event in events %}
+                        <tr class = "click-row" data-href="modifyEvent/ + {{event.id}}">
+                            <td>{{event.name}}</td>
+                            <td>{{event.planner.name}}</td>
+                            <td>{{event.time.date()}}</td>
                         </tr>
+                        {% endfor %}
                     </tbody>
                 </table>
             </div>
                 <div class="buttons-action">
-                <button class="btn btn-lg btn-primary btn-add" type="submit">Add</button>
+                <button id="addEvent" class="btn btn-lg btn-primary btn-add" type="button">Add</button>
             </div>
         </div>
 <script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
 <script src="/static/bootstable.js"></script>
 
 <script>
-    $('table').SetEditable();
+    $('table').SetEditable({editButton: false, onBeforeDelete: deleteEvent});
+
+    window.onload = onDocumentReload;
+
+    function onDocumentReload() {
+        url = window.location.href;
+        parameter = url.split("?");
+        if (parameter[1] == "show_all=yes") {
+            document.getElementById("showAllCB").checked = true;
+        }
+    }
+
+    function refresh() {
+       url = window.location.href;
+       window.location = url;
+    }
+
+    $(".click-row").click(function() {
+        window.location = $(this).data("href");
+    });
+
+    $("#addEvent").click(function() {
+        window.location.href = "/modifyEvent";
```

Comment:
I designed mine with two different functions. This works if you change this to:

    $("#addEvent").click(function() {
        window.location.href = "/addEvent";
    });